src/quo2/components/numbered_keyboard/numbered_keyboard/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.numbered-keyboard.numbered-keyboard.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.numbered-keyboard.keyboard-key.view :as keyboard-key]
+            [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.style :as style]))
+
+(defn keyboard-item
+  [{:keys [item type disabled? on-press blur? theme]
+    :or   {item nil}}]
```

Comment:
If `item` is not provided, it already is `nil`, so we could remove that `:or` destructuring.

Reply:
Done